src/state/contents/actions.js

```diff
@@ -100,17 +120,80 @@ export const fetchContents = (page = pageDefault,
           json.errors.forEach(err => dispatch(addToast(err.message, TOAST_ERROR)));
           dispatch(clearErrors());
         }
+        dispatch(selectAllRows(false));
         dispatch(toggleLoading('contents'));
         resolve();
       });
     })
     .catch(() => {});
 });
 
-export const fetchContentsPaged = () => (dispatch, getState) => {
-  const pagination = getPagination(getState(), 'contents');
-  // @ TODO get filter properties and convert to query string and add as param
-  return dispatch(fetchContents(pagination));
+export const fetchContentsWithFilters = (
+  params, newPagination, newSort,
+) => (dispatch, getState) => {
+  const state = getState();
+  const pagination = newPagination || getPagination(state, 'contents') || getPagination(state);
+  const sortingColumns = getSortingColumns(state);
+  const quickFilter = getCurrentQuickFilter(state);
+  const { id, value: qfValue } = quickFilter;
+  const columnKey = Object.keys(sortingColumns)[0];
+  const sortDirection = sortingColumns[columnKey].direction;
+  const sortParams = newSort || `sort=${columnKey}&direction=${sortDirection.toUpperCase()}`;
+  let filterParams = '';
+  const filters = [];
+  if (params) { filterParams = params; return dispatch(fetchContents(pagination, `${params}&${sortParams}`)); }
+  if (qfValue) {
+    filters.push({ att: id, value: qfValue });
+  } else {
+    const contentType = getContentType(state);
+    const group = getGroup(state);
+    const filteringCategories = getFilteringCategories(state);
+    const status = getStatusChecked(state) || getCurrentStatusShow(state);
+    const access = getAccessChecked(state);
+    const author = getAuthorChecked(state) || getCurrentAuthorShow(state);
+    if (contentType) filters.push({ att: 'typeCode', value: contentType });
+    if (group) filters.push({ att: 'mainGroup', value: group });
+    if (status) filters.push({ att: 'status', value: status });
+    if (access) filters.push({ att: 'access', value: access === 'free' ? 'free' : 'closed' });
+    if (author && author !== 'all') filters.push({ att: 'firstEditor', value: author });
+    if (filteringCategories && filteringCategories.length) filters.push({ att: 'categories', value: filteringCategories });
+  }
+  filterParams = filters.map(({ att, value }, i) => {
+    if (att === 'categories') {
+      return value.map(
+        (filter, j) => `&filters[${i + j}].attribute=categories&filters[${i + j}].value=${filter.code}`,
+      ).join('');
+    }
+    if (att === 'status' && value === 'published') {
+      return '&status=published';
+    }
+    return `&filters[${i}].attribute=${att}&filters[${i}].value=${value}`;
+  }).join('');
```

Comment:
you can create all these filtering parameters with this existing tool in entando frontend-libraries (entando-utils). Please check starting at this point https://github.com/entando/frontend-libraries/tree/master/packages/entando-utils#addfilterfilter-string to know about the methods and please utilizing those including future filtering components. I have used these methods in assets/actions, feel free to check them.